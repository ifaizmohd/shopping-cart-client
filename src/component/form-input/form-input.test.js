import { shallow } from "enzyme";
import React from "react";
import FormInput from "./form-input.component";

describe("Form input component testing", () => {
  it("Shallow snapshot testing of form input component", () => {
    expect(shallow(<FormInput />)).toMatchSnapshot();
  });
});
