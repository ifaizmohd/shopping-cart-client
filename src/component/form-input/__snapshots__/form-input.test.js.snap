// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form input component testing Shallow snapshot testing of form input component 1`] = `ShallowWrapper {}`;
